harambe/errors.py

```diff
@@ -0,0 +1,16 @@
+from harambe.types import ScrapeResult, Schema
+
+
+class SchemaValidationError(Exception):
+    def __init__(self, schema: Schema, data: ScrapeResult, message: str):
+        super().__init__(
+            "Data {data} does not match schema {schema}. {message}".format(
+                data=data, schema=schema, message=message
+            )
+        )
+
+
+class CaptchaError(Exception):
+    def __init__(self, message="CAPTCHA was hit."):
+        self.message = message
+        super().__init__(self.message)
```

Comment:
We should make both of these extend HarambeException so we can catch them easily